crates/typst/src/geom/angle.rs

```diff
@@ -66,10 +68,32 @@ impl Angle {
         self.to_rad().cos()
     }
 
+    /// Get the sine and cosine of this angle in radians.
+    pub fn sin_cos(self) -> (f64, f64) {
+        self.to_rad().sin_cos()
+    }
```

Comment:
The std impl is literally `(self.sin(), self.cos())`, so it doesn't seem to be a perf thing. Which makes me wonder whether this is really worth the few characters less. It seems like a somewhat strange abstraction to me.

Reply:
I agree, my best guess is that it helps the compiler optimize better on some targets? I like the slightly shorter nature, but it doesn't do much apart from looking slightly nicer.

Reply:
I would remove it.